server/src/test/java/org/elasticsearch/index/shard/IndexShardIT.java

```diff
@@ -273,6 +273,8 @@ public void testExpectedShardSizeIsPresent() throws InterruptedException {
         assertTrue(test > 0);
     }
 
+    // NORELEASE This test need to be adapted for replicated closed indices
+    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/33888")
```

Comment:
is this test mentioned somewhere explicitly on the meta issue? add a new bullet point?

Reply:
Sorry it's not. I'll add a bullet point to #33888. The test will be fixed in a follow up pull request (code is ready)